### ISSUE TITLE
Unable to open app or view menu via Tray Notifier

### ISSUE THREAD

**Issue by Bilibox** [safing/portmaster-ui#232]
**What happened**:

Nothing pops up when LMB/RMB is pressed.

**What did you expect to happen?**:

A menu to show.

**How did you reproduce it?**:

Left or right click on the tray notifier

**Comment by dhaavi** [comment]
Hey @Bilibox, thanks for raising this issue.

Can you tell us which OS (+version) you are using?

**Comment by Bilibox** [comment]
Hey @dhaavi , I'm currently using Windows 11 Pro 21H2 build 22000.613

**Comment by Kiefy** [comment]
Windows 10 here.

It is rare, but I'd say it's happened to me at least 20 times.  I have to kill the notifier in task manager to fix it.

**Comment by nik-field** [comment]
Win10 and I was recently having issues with the tray icon not working at all. I was also working on another issue in parallel where Snip & Sketch would not save my screenshots. After going through a clean boot (disabling/enabling services, restarting, etc) I was able to identify that the Nvidia LocalSystem Container Service was the issue for Snip & Sketch. 

After some more brainstorming/troubleshooting I disabled Nvidia GeForce Experience's In-Game Overlay. This fixed Snip & Sketch and the Tray Icon context menu now works as expected.

Not sure if you have a similar setup or if this will fix your issue but it worked for me!
